tests/WorkflowRun.py

```diff
@@ -29,83 +29,105 @@ class WorkflowRun(Framework.TestCase):
     def setUp(self):
         super().setUp()
         self.repo = self.g.get_repo("PyGithub/PyGithub")
-        self.workflow_run = self.repo.get_workflow_run(148274629)
+        self.workflow_run = self.repo.get_workflow_run(3881497935)
 
     def testAttributes(self):
         self.assertEqual(
             repr(self.workflow_run),
-            'WorkflowRun(url="https://api.github.com/repos/PyGithub/PyGithub/actions/runs/148274629", id=148274629)',
+            'WorkflowRun(url="https://api.github.com/repos/PyGithub/PyGithub/actions/runs/3881497935", id=3881497935)',
         )
-        self.assertEqual(self.workflow_run.id, 148274629)
-        self.assertEqual(self.workflow_run.head_branch, "more-precise-typing")
+        self.assertEqual(self.workflow_run.id, 3881497935)
+        self.assertEqual(self.workflow_run.name, "CI")
+        self.assertEqual(self.workflow_run.head_branch, "feat/workflow-run")
         self.assertEqual(
-            self.workflow_run.head_sha, "f91c729d786efcc93db47dd755313a26172c105e"
+            self.workflow_run.head_sha, "c6e5cac67a58a4eb11f1f28567a77a6e2cc8ee98"
         )
-        self.assertEqual(self.workflow_run.run_number, 162)
+        self.assertEqual(self.workflow_run.path, ".github/workflows/ci.yml")
+        self.assertEqual(self.workflow_run.display_title, "TEST PR")
+        self.assertEqual(self.workflow_run.run_number, 930)
```

Comment:
While we are here, can we also assert existing `run_attempt` field?
```suggestion
        self.assertEqual(self.workflow_run.run_attempt, ...)
        self.assertEqual(self.workflow_run.run_number, 930)
```